### DISCUSSION TITLE
Any way i can use centralise location to refer provider.tf for customise plugin location?

### DISCUSSION THREAD

**Discussion by yogeshlc** [opentofu/opentofu#1430] [Q&A]
I want to use custom plugin & I have 5 different terrafrom (opentofu) resource template placed at different location. But i dont want to refer the provider.tf for all the resources every time i am doing "tofu init".
I tried using .terraformrc to provide the custom location for plugin but it seems not working.

Any idea?

**Comment by ghost** [comment]
Hey @yogeshlc unfortunately, it's very hard to help without seeing the code. You may want to create a small reproduction project and upload it as a ZIP or put it in a git repo.
